SoftwareDesignDocument..txt

```diff
@@ -67,5 +67,19 @@ Code:
 Req 1.2.1
 	Add new block to item.h and item.cpp
 Code:        
-	#define TANK_SHELL 64 in the item.h
+  #define TANK_SHELL 64 in the item.h
 	TANK_SHELL In items[] in the item.cpp file
+
+Req 1.3, 1.3.1, 1.3.3
+	When the user types "/play tank minigame" into the command, spawn the user into an arena at a fixed location.
+Code: 
+	//this function will get called upon "/play tank minigame" being inputed into the command line
+	void gameStart()
```

Comment:
This function looks good, the only thing missing is equipping both the plater tank and the enemy tank with the projectile blocks unless you have another way of implementing that.